desktop/faqs.md

```diff
@@ -81,6 +81,18 @@ nested virtualization scenarios**. It might work in some cases, and not in other
 
 For more information, see [Running Docker Desktop in nested virtualization scenarios](../docker-for-windows/troubleshoot.md#running-docker-desktop-in-nested-virtualization-scenarios).
 
+### Docker Desktop's UI appears green, distorted, or has visual artifacts. How do I fix this?
+
+Docker Desktop is built on Electron which enables hardware-accelerated graphics by default. 
+However, some graphics cards may not support hardware acceleration. In such cases, 
+Docker Desktop will launch successfully, but some screens may appear green, distorted, 
+or have some visual artifacts.
+
+To work around this issue, disable hardware acceleration by 
+setting the `DOCKER_DESKTOP_DISABLE_HARDWARE_ACCELERATION` environment 
+variable to any value. For example, set `DOCKER_DESKTOP_DISABLE_HARDWARE_ACCELERATION=1` 
+and then restart Docker Desktop.
```

Comment:
@trungutt We should probably have some instructions about how to do this? E.g. a Powershell snippet for Windows.